src/shared/locales/en/translation.json

```
@@ -594,6 +596,23 @@
         "nodeChangeError": "Error changing node",
         "nodeChangeErrorExplanation": "There was an error changing node. Please try again."
     },
+    "sweeps": {
+        "recoverLockedFunds": "Recover locked funds",
+        "recover": "Recover",
+        "pressButtonBelow": "Press the button below to recover your locked funds.",
+        "lockedFundsDetected": "Trinity has detected locked funds in your account.",
+        "lockedFundsRecovery": "Locked Funds Recovery",
+        "aboutLockedFunds": "About Locked Funds",
+        "lockedFundsRecoveryExplanation": "Funds have been detected on one or more addresses that have already been spent from. Spending from an address more than once is insecure.",
+        "lockedFundsToolExplanation": "This tool will attempt to salvage those funds in a way that reduces the risk involved in spending more than once. Recovery cannot be guaranteed.",
+        "lockedFundsOwnRiskExplanation": " Use this tool at your own risk.",
```

```suggestion
        "lockedFundsOwnRiskExplanation": "Use this tool at your own risk.",
```
Extra space at the beginning of the string